src/daemon/index.ts

```diff
@@ -174,7 +191,7 @@ async function traceLSPPerformance(javaExt: vscode.Extension<any>) {
       }
 
       if (traceEvent.resultLength === 0) {
```

Comment:
How about the isEmpty check to also cover empty objects?

Reply:
resultLength is not the raw response object. It's a processed value parsed from the response.

See
https://github.com/redhat-developer/vscode-java/blob/812613b5923ba27508f0a15a605f4cf80faa2388/src/TracingLanguageClient.ts#L134